src/lib/linting.js

```diff
@@ -171,14 +171,14 @@ class Linting extends EventEmitter {
         for (let k in this.results) {
             const result = this.results[k];
             if (result instanceof Array) {
-                if (result.some(res => res.hasErrors)) {
+                if (result.some(res => res.hasErrors) || result.some(res => res.hasExceptions)) {
```

Comment:
Probably best to use `result.some(res => res.hasErrors || res.hasExceptions)`, so we don't loop the array twice.